client/dashboard/profile-wizard/steps/theme/uploader.js

```diff
@@ -0,0 +1,117 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import classnames from 'classnames';
+import { Component, Fragment } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { DropZoneProvider, DropZone, Spinner } from '@wordpress/components';
```

Comment:
We should use WooCommerce Admin's Muriel `Spinner` component (the one used on plugin install pages).

Reply:
Updated in https://github.com/woocommerce/woocommerce-admin/pull/2580/commits/bea273758a8e60f3700ca9ddf1c94f578e9e7b7e